src/components/Pages/HomePage.js

```diff
@@ -1,20 +1,23 @@
 import React, { useEffect, useState, useContext } from 'react';
-import { getHighlyRatedEpisodes } from '../../api/api';
+import { TvPosterSkeleton } from '../Main/TvPosterSkeletonLoading';
 import { TVShowPoster } from '../Main/TvShowPoster';
 import { SeriesSearchForm } from '../SeriesSearch/SeriesSearchForm';
 import { AppContext } from '../../contexts/AppContext';
+import { getHighlyRatedEpisodes } from '../../api/api';
 import { _ } from '../../utils';
 
 // http://localhost:8080/
 export const HomePage = () => {
   const { setIsLoading } = useContext(AppContext);
-  const [episodesData, setEpisodesData] = useState([]);
+  const [tvPosterData, setTvPosterData] = useState([]);
+
+  const PREVIEW_SERIES_COUNT = 5;
```

Comment:
should move to constants 

https://github.com/starter-code/binge/blob/master/src/constants/constants.js